tests/test_data.py

```diff
@@ -36,6 +37,16 @@
         "returns_srcid": "S1A_IW_SLC__1SDV_20200224T052528_20200224T052555_031390_039CF2_BEA6",
         "returns_uuid": "8a611d5b-f9d9-437e-9f55-eca18cf79fd4",
     },
+    {
+        "datahub": Datahub.Scihub,
+        "datadir": None,
+        "platform_name": Platform.Sentinel1,
+        "date": ("20200502", "20200503"),
+        "aoi": aoi_wkt,
+        "cloud_cover": None,
+        "returns_srcid": "S1A_IW_GRDH_1SDV_20200502T170726_20200502T170751_032389_03BFFF_3105 ",

```

Comment:
The test fails here, because the return value contains a space that should not be there.